--- source-highlight-2.8/doc/source-highlight.texinfo.orig	2007-09-09 16:45:17.000000000 +0200
+++ source-highlight-2.8/doc/source-highlight.texinfo	2007-11-30 22:37:15.403856539 +0100
@@ -32,9 +32,9 @@
 @end quotation
 @end copying
 
-@dircategory Utilities
+@dircategory Text Processing Tools:
 @direntry
-* Source-highlight: (source-highlight).  Highlights contents
+* Source-highlight: (source-highlight).	Highlights contents
 @end direntry
 
 @titlepage
